### ISSUE TITLE
Run as a module 

### ISSUE THREAD

**Issue by olawalejuwonm** [ChetanXpro/autosub#34]
Is there a possibility of running this as part of application without necessarily using it as a global cli program?

Great work 👏 overall

**Comment by ChetanXpro** [comment]
Yes, that's a great idea! I will create a library that you can easily import into your existing project. Most likely, I will maintain this project as a CLI tool and create a separate project for the library.

**Comment by olawalejuwonm** [comment]
@ChetanXpro oh great. I will be looking forward to that
